### ISSUE TITLE
[Feature] : blindness / visual impaired  Accessibility

### ISSUE THREAD

**Issue by kaminosekai54** [pagefaultgames/pokerogue#1858]
**Is your feature request related to a problem? Please describe.**
As a visually impaired player (Who really enjoy the game btw), I often struggle to get the infosrmations, I'm using a screen reader (NVDA) and to get the informations, I have to use the OCR functionnality of it, which struggle a lot. and is limited.
For example, I can not know which starter have the pokerus, how many hp my poké lost, or have, the nature, move set of captured pokémon, the IVS neither, etc, etc.

**Describe the Feature**
The idea is simple, add screen reader compatibility, since controler controle is activated, vocalise the selected item, poké, event.
As a good example, the game, slay the spire, have a mod say the spire that do this kind of things.
To be more clear.
In the starter select screen, annonce the selected pokemon, it's nature / passive, number of candy, iv's etc, etc, annonce when nature or ability is changed, when shiny state is changed, with the luck.
Basically, when the cursor move, vocalise the news informations, in menu, battle, etc, etc.
